src/test/java/uk/nhs/digital/nhsconnect/nhais/mapper/PersonDateOfBirthMapperTest.java

```diff
@@ -1,35 +1,31 @@
 package uk.nhs.digital.nhsconnect.nhais.mapper;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertThrows;
+import org.hl7.fhir.r4.model.Parameters;
+import org.hl7.fhir.r4.model.Patient;
+import org.junit.jupiter.api.Test;
+import uk.nhs.digital.nhsconnect.nhais.model.edifact.PersonDateOfBirth;
+import uk.nhs.digital.nhsconnect.nhais.service.edifact_to_fhir.PatientParameter;
 
 import java.sql.Date;
 import java.time.Instant;
 import java.time.ZoneId;
 import java.time.ZonedDateTime;
 
-import uk.nhs.digital.nhsconnect.nhais.model.edifact.PersonDateOfBirth;
-import uk.nhs.digital.nhsconnect.nhais.service.edifact_to_fhir.PatientParameter;
-
-import org.hl7.fhir.r4.model.Parameters;
-import org.hl7.fhir.r4.model.Patient;
-import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
 
 class PersonDateOfBirthMapperTest {
     private static final Instant FIXED_TIME = ZonedDateTime.of(
-        1991,
-        11,
-        6,
-        23,
-        55,
-        0,
-        0,
-        ZoneId.of("Europe/London")).toInstant();
+        1991, 11, 6, 23, 55, 0, 0, ZoneId.of("Europe/London"))
+        .toInstant();
+    private static final Instant FIXED_TIME_LOCAL = ZonedDateTime.of(
+        1991, 11, 6, 23, 55, 0, 0, ZoneId.systemDefault())
```

Comment:
Why did you change this to systemDefault? Will we get a different results depending on the system's time zone?

Reply:
It won't. This is needed to pass tests in non-UK zone as fhir parser takes local zone